core/itwin-connector-framework/src/iTwinConnector.ts

```diff
@@ -9,62 +9,62 @@
 import { assert, BentleyStatus, ClientRequestContext } from "@bentley/bentleyjs-core";
 import { Subject } from "@bentley/imodeljs-backend";
 import { AuthorizedClientRequestContext } from "@bentley/itwin-client";
-import { BridgeJobDefArgs } from "./BridgeRunner";
+import { ConnectorJobDefArgs } from "./ConnectorRunner";
 import { Synchronizer } from "./Synchronizer";
 
-/** Abstract implementation of the IModelBridge.
+/** Abstract implementation of the iTwin Connector.
  * @beta
  */
-export abstract class IModelBridge {
+export abstract class ItwinConnector {

```

Comment:
This should probably be `ITwinConnector` with the capital T.

Reply:
Done.